C/casacore/build_tarballs.jl

```diff
@@ -0,0 +1,109 @@
+platforms = supported_platforms(exclude=Sys.iswindows)
+# Deal with the fact that we have std::string values, which causes issues across the gcc 4/5 boundary
+platforms = expand_cxxstring_abis(platforms)
+# expand gfortran versions as well
+platforms = expand_gfortran_versions(platforms)
+
+# The products that we will ensure are always built
+products = Product[
+    LibraryProduct("libcasa_casa", :libcasa_casa),
+    LibraryProduct("libcasa_coordinates", :libcasa_coordinates),
+    LibraryProduct("libcasa_derivedmscal", :libcasa_derivedmscal),
+    LibraryProduct("libcasa_fits", :libcasa_fits),
+    LibraryProduct("libcasa_images", :libcasa_images),
+    LibraryProduct("libcasa_lattices", :libcasa_lattices),
+    LibraryProduct("libcasa_meas", :libcasa_meas),
+    LibraryProduct("libcasa_measures", :libcasa_measures),
+    LibraryProduct("libcasa_mirlib", :libcasa_mirlib),
+    LibraryProduct("libcasa_msfits", :libcasa_msfits),
+    LibraryProduct("libcasa_ms", :libcasa_ms),
+    LibraryProduct("libcasa_scimath_f", :libcasa_scimath_f),
+    LibraryProduct("libcasa_scimath", :libcasa_scimath),
+    LibraryProduct("libcasa_tables", :libcasa_tables),
+    ExecutableProduct("findmeastable", :findmeastable),
+    ExecutableProduct("fits2table", :fits2table),
+    ExecutableProduct("image2fits", :image2fits),
+    ExecutableProduct("imagecalc", :imagecalc),
+    ExecutableProduct("imageregrid", :imageregrid),
+    ExecutableProduct("imageslice", :imageslice),
+    ExecutableProduct("lsmf", :lsmf),
+    ExecutableProduct("measuresdata", :measuresdata),
+    ExecutableProduct("measuresdata-update", :measuresdata_update),
+    ExecutableProduct("ms2uvfits", :ms2uvfits),
+    ExecutableProduct("msselect", :msselect),
+    ExecutableProduct("readms", :readms),
+    ExecutableProduct("showtableinfo", :showtableinfo),
+    ExecutableProduct("showtablelock", :showtablelock),
+    ExecutableProduct("tablefromascii", :tablefromascii),
+    ExecutableProduct("taql", :taql),
+    ExecutableProduct("tomf", :tomf),
+    ExecutableProduct("writems", :writems),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
+    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"))
+    Dependency(PackageSpec(name="CFITSIO_jll", uuid="b3e40c51-02ae-5482-8a39-3ace5868dcf4"))
+    Dependency(PackageSpec(name="WCS_jll", uuid="550c8279-ae0e-5d1b-948f-937f2608a23e"))
+    Dependency(PackageSpec(name="Readline_jll", uuid="05236dd9-4125-5232-aa7c-9ec0c9b2c25a"))
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.7", preferred_gcc_version = v"5.2.0")
```

Comment:
The problem with Musl should be solved by using GCC 6
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.7", preferred_gcc_version = v"6")
```